jsapp/scss/components/_kobo.form-view.scss

```diff
@@ -459,6 +459,19 @@
     }
   }
 
+  &.form-view__cell--error {
+    @extend .form-view__cell--warning;
```

Comment:
Please [avoid @extend](https://www.sitepoint.com/avoid-sass-extend/). Either write shared code by list of selectors (`&.form-view__cell--warning, &.form-view__cell--warning {}`) or duplicate it :P

Reply:
Interesting article :) we have a lot to clean up